src/localization/i18n.ts

```diff
@@ -0,0 +1,18 @@
+import i18n from 'i18next';
+import {initReactI18next} from 'react-i18next';
+import en from './translations/en.json';
+import ua from './translations/ua.json';
+import ru from './translations/ru.json';
+
+const resources = {
+  en,
+  ua,
+  ru,
+};
+
+export const localization = i18n.use(initReactI18next).init({
```

Comment:
its better to wrap this library with some class and use i18n in class only, you can setup Localisation class for example and override t func + change language etc that you actually need
And other your code don't know that you are using i18n .... that cool because we dont have such dependency directly to specific library :)